ansible/roles/ml-analytics-scripts/templates/config.j2

```diff
@@ -0,0 +1,109 @@
+[dev]
+host = "{{proto}}://{{domain_name}}/" 
+internal_kong_ip = "http://{{internal_kong_ip}}:8000" 
+keyclockapibody = client_id={{sunbird_sso_client_id}}&client_secret={{sunbird_sso_client_secret | d('')}}&grant_type=client_credentials&scope=offline_access
```

Comment:
@raghupathiguduri  can you please check this line number 4. It is not enclosed with double quotes. Did you verify that it is able construct the string?